contentcuration/contentcuration/frontend/channelList/views/Channel/CatalogList.vue

```diff
@@ -59,7 +59,7 @@
           </VLayout>
         </VFlex>
       </VLayout>
-      <BottomToolBar
+      <BottomBar
```

Comment:
Hi! @cerdo03. We need to be careful because the implementation of BottomToolBar and BottomBar are slightly different, for example BottomBar does not recieve any props, but BottomToolBar does. So all places where BottomToolBar receives props are likely to have errors. I let you an example for the `CalatogList` component:

Before the change:
![image](https://github.com/learningequality/studio/assets/51239030/c7b853d3-b732-4fe0-9b96-a343be4d3939)

After the change:
![image](https://github.com/learningequality/studio/assets/51239030/7c44f5c1-ed3c-4c17-9d56-7243b104d5de)

The views should remain the same after the change to avoid unintended behaviours.